module/move/wca/src/ca/formatter.rs

```diff
@@ -3,6 +3,7 @@ pub( crate ) mod private
 
   use crate::*;
   use wtools::Itertools;
+  use crate::ca::aggregator::private::Order;
```

Comment:
This seems to be not in line with the [convention](https://github.com/obox-systems/conventions/blob/master/code_style.md#importing-local-entities)